### DISCUSSION TITLE
Support for vision / images?

### DISCUSSION THREAD

**Discussion by ccpking112** [microsoft/prompty#73] [Q&A]
What support does prompty currently provide for vision / image analysis?  (Open AI, for example, supports analysis of images passed as URLs, local paths, or base64 encoded data.  I would guess that prompty now supports image analysis, but I haven't been able to find any documentation showing how.)

**Comment by sethjuarez** [comment]
Hi! Currently the parser creates the proper image resources for vision based LLM's to do their best. It should totally work!
